kong/plugins/influxdb-log/handler.lua

```diff
@@ -0,0 +1,98 @@
+local HttpLogHandler = require "kong.plugins.http-log.handler"
+local cjson = require "cjson"
+
+local InfluxdbLogHandler = HttpLogHandler:extend()
+
+InfluxdbLogHandler.PRIORITY = 1
+
+-- influxdb util function
+-- ref:https://docs.influxdata.com/influxdb/v0.13/write_protocols/write_syntax/
+local function influxdb_point(ngx)
```

Comment:
considering the amount of table lookups, this function would benefit from caching the tables